components/cli/pkg/util/utils.go

```diff
@@ -584,22 +584,19 @@ func ExecuteCommand(cmd *exec.Cmd, errorMessage string) error {
 	}()
 	err := cmd.Start()
 	if err != nil {
-		fmt.Printf("cellery : %v: %v \n", errorMessage, err)
-		os.Exit(1)
+		ExitWithErrorMessage("Error executing command", err)
```

Comment:
Shall we add the command name to the error message?

Reply:
Returned the error instead of exiting with error message.